### PR TITLE
Drop report feature

### LINKED ISSUE #310: AE: Parse drop report and correlate to determine when attack has stopped.
Display number of dropped packets.

### PR DESCRIPTION
**Pull request by akadam**
Parses drop report, computes sha256 hash for correlation to drop report.
#### Questions:
- Have you connected this PR to the issue it resolves? Yes
- Does the documentation need an update? No
- Does this add new dependencies? No
- Have you added unit or functional tests for this PR? No

### CODE REVIEW

**Comment by akadam** [comment]
Run Tofino Environment Build

**Comment by akadam** [comment]
Run Tofino lab trial build

File: playbooks/ae/templates/drop_report_hash_script.json [hunk 1]
```diff
@@ -15,7 +15,7 @@ int[] HS() {
   };
 }
 int[] KS() {
-  return new int[] {
+return new int[] {
```
Comment (spisarski) [inline playbooks/ae/templates/drop_report_hash_script.json:18]: What are these hard-coded int values?

**Review by spisarski — APPROVED** [review]
My only real question I should have asked earlier is about the hard-coded list of long int values, but it is all good until we find a bug :)

**Pull request merged**